### PR TITLE
Fixes require constraint so Composer on Windows can actually resolve correct version between Testbench and Laravel Framework

### PR DESCRIPTION
**Pull request by crynobone**
https://github.com/crynobone/laravel-value-objects/actions/runs/3142867966/jobs/5106962945

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -40,7 +37,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
+          composer require "laravel/framework:${{ matrix.laravel }}" --dev --no-interaction --no-update
```
Comment (crynobone) [inline .github/workflows/run-tests.yml:40]: Testbench is already installed with `^7.4` as the minimum version matching Laravel Framework 9.7. Requiring it here messes with Composer on Windows.

File: composer.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": "^8.0",
+        "illuminate/support": "^9.7",
```
Comment (crynobone) [inline composer.json:20]: It's always better to lock the minimum supported version either using `illuminate/support` or `illuminate/contract` under `require`

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (crynobone) [reply .github/workflows/run-tests.yml:40]: Unless you are testing multiple versions of Laravel, this line isn't even necessary.

**Comment by michael-rubel** [comment]
Thank you for figuring it out! I'll update the package template accordingly.

**Pull request merged**